lib/tests/systems.nix

```diff
@@ -19,6 +19,13 @@ let
 
     NOTE: This property is not guaranteed when `sys` was elaborated by a different
           version of Nixpkgs.
+
+
+    # Inputs
+
+    `sys`
+
+    : 1\. Function argument
```

Comment:
Given that this is back to a normal comment, the extra info there doesn't help at all because it won't be rendered anywhere else.

```suggestion
```